### ISSUE TITLE
Windows 7. Saving the options does not persist the setting, no write error message due to access control.

### ISSUE THREAD

**Issue by JVimes** [dnGrep/dnGrep#70]
_From [rajs...@gmail.com](https://code.google.com/u/114329306507443562799/) on April 09, 2010 10:28:27_

Open dngrep->Options->Editor (Change to custom editor say notepad++)
Save.
The setting is good for the session.
Close.
Open again and the editor setting is not saved.
Issue, windows 7 does not allow dngrep to create dnGREP.Settings.dat.

Workaround.
Open dngrep as administrator (RMB->Run as Administrator)
Save the setting and close.
Next time onwards the setting will work fine as the file 
(dnGREP.Settings.dat) is saved, for any user. 

Basically dngrep should report an error message if it fails to save the 
setting dat file.

_Original issue: http://code.google.com/p/dngrep/issues/detail?id=66_
